compareECLFiles.cmake

```diff
@@ -312,7 +314,8 @@ add_test_compareECLFiles(CASENAME faults_model_1
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
                          REL_TOL ${rel_tol}
-                         DIR model1)
+                         DIR model1
+                         TEST_ARGS --tolerance-pressure-ms-wells=10. --tolerance-wells=1.e-6 --solver-max-time-step-in-days=5.0)
```

Comment:
Same here, great to see this one recovered, but are we sure we need more that just the time step constraint?